testing/mmc-utils/APKBUILD

```diff
@@ -1,30 +1,30 @@
 # Contributor: Olliver Schinagl <oliver@schinagl.nl>
 # Maintainer: Olliver Schinagl <oliver@schinagl.nl>
 pkgname=mmc-utils
-pkgver="4.3.0_git20181024"
-pkgrel=0
+pkgver="0_git$(date +%Y%m%d)"
```

Comment:
The package version should be static. If the same package with the same source is built, it should have the same version. Now it will get a new version if it's built on a different day.

Reply:
Ah I think this is a leftover from when I didn't store the commitish, but just pulled master. So each time you build, you pull a new master, so the date would be new, everytime you build, you get a new hash e.g. new version.

Would be much nicer if apk would support commitish in its version field ...

Reply:
The version field is used to determine whether a package should be updated. A random hash is not beneficial for that.

Reply:
yes of course, but that does not make it less useful :p you want the date to be incremental, rather then the hash; that's completely understandable but the date also only tells you so much, anyway, that would be a generic alpine discussion, for a different thread. I've updated the PR as requested ;)